apps/server/cypress/e2e/fuzzySearch.cy.ts

```diff
@@ -1,16 +1,25 @@
 describe('Should be able to find certain banks with keywords and misspellings', () => {
-  it('Finds expected banks', () => {
+  beforeEach(() => {
```

Comment:
Abstracting this is fine, but I don't think it adds a lot of value. It requires me to look outside of the tests for a single line of code that's easy to repeat